crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -45,6 +45,10 @@ use self::reconstruction::FragmentReconstruction;
 pub type DBLabel = usize;
 const LOCALS: DBLabel = 0;
 
+// TODO: Make last checkpoint number of each epoch more flexible.
+// TODO: Make this bigger.
+pub const CHECKPOINT_COUNT_PER_EPOCH: u64 = 3;
```

Comment:
Lets make that 10. I don't know why but makes me feel safer :)

Reply:
Don't worry, we will certainly make this at least 10 when we actually turn on reconfig. Setting it to 3 makes it faster/easier to test locally.